src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -2359,9 +2359,8 @@ protected override void HandleCompletionsRequestAsync(IRequestResponder<Completi
                 command = command.Substring(6);
             try
             {
-                MethodInfo methodInfo = m_program.GetType().GetMethod("AutoComplete");
-                string[] results = (string[])methodInfo.Invoke(m_program, new object[] { command, frame });
-                if (results == null)
+                var debugProgram = (IDebugProgramDAP)m_engine;
```

Comment:
For some reason this fails for me locally while the following works
```cs
var methodInfo = m_engine.GetType().GetInterface("IDebugProgramDAP").GetMethod("AutoCompleteCommand");
```

Reply:
What OS are you on?

Reply:
WSL. The DAP dll isn't part of the vscode plugin distribution so I tried copying it but to no avail.

Reply:
It looks like an issue with the mono we ship, it does not handle [`EmbedInteropTypes` correctly](https://github.com/microsoft/MIEngine/blob/fbbc9938f8a75e3ca5fb7573a331f61ff484676f/src/OpenDebugAD7/OpenDebugAD7.csproj#L93).

I'll push out a fix to deploy it if we are in the XPlat scenario.

Reply:
@WardenGnaw did that happen already?

Reply:
Apparently not, so let's hope for the .net 5 change to fix it.

Reply:
The work to switch to running on .NET 5 is almost complete.

Reply:
Ok it still fails.
```cs
m_engine is IDebugProgramDAP
error CS0433: The type 'IDebugProgramDAP' exists in both 'Microsoft.MIDebugEngine, Version=14.0.0.0, Culture=neutral, PublicKeyToken=cfb98189dc466aca' and 'OpenDebugAD7, Version=14.0.0.0, Culture=neutral, PublicKeyToken=cfb98189dc466aca'
```
Needed to disable Embed and copy the DAP dll manually.